ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -10,6 +10,37 @@ public protocol BottomCommandingControllerDelegate: AnyObject {
 
     /// Called when `collapsedHeightInSafeArea` changes.
     @objc optional func bottomCommandingControllerCollapsedHeightInSafeAreaDidChange(_ bottomCommandingController: BottomCommandingController)
+
+    /// Called after the bottom sheet expansion state changes.
+    ///
+    /// External changes to `isHidden` will not trigger this callback.
+    /// - Parameters:
+    ///   - bottomCommandingController: The caller object.
+    ///   - expansionState: The expansion state the sheet moved to.
+    ///   - commandingInteraction: If the state change was caused by user interaction, it will be indicated using this enum.
+    ///   - sheetInteraction: If `commandingInteraction` is `.sheetInteraction`, this enum will contain more information about what triggered the state change.
+    @objc optional func bottomCommandingController(_ bottomCommandingController: BottomCommandingController, sheetDidMoveTo expansionState: BottomSheetExpansionState, commandingInteraction: BottomCommandingInteraction, sheetInteraction: BottomSheetInteraction)
+
+    /// Called after the bottom bar popover is presented.
+    /// - Parameters:
+    ///   - bottomCommandingController: The caller object.
+    ///   - commandingInteraction: The user interaction that caused the popover to show.
+    @objc optional func bottomCommandingControllerDidPresentPopover(_ bottomCommandingController: BottomCommandingController, commandingInteraction: BottomCommandingInteraction)
```

Comment:
minor: like above when there are multiple parameters I *think* convention is to make it `bottomCommandingController(bottomCommandingController: BottomCommandingController, didPresentPopoverFromInteraction:...`

Reply:
I think it should even be:
bottomCommandingController(_ bottomCommandingController: BottomCommandingController, didPresetPopoverFrom interaction: ...
It seemed pretty horrible to me, but you're right, that's the convention.

Reply:
Never mind, it's actually not that bad at all!